mopidy_tunein/tunein.py

```diff
@@ -360,6 +368,9 @@ def search(self, query):
     @cache()
     def _tunein(self, variant, args):
         uri = (self._base_uri % variant) + '?render=json' + args
+        # if config filters are set add it
+        if (self._filter is not None):
```

Comment:
We don't need brackets around any of these if-statements. I'm surprised flake8 likes that.